packages/v4/patternfly-docs/content/design-guidelines/components/data-list/data-list.md

```diff
@@ -21,7 +28,14 @@ Think of each row in a data list as a container for some formatted content. In P
 * **[Split](/layouts/split/react):** Distributes content evenly with a main content area in the center.
 * **[Flex](/layouts/flex/react):** Enables more customization control over the alignment and spacing options provided in the other layouts.
```

Comment:
It looks like all the "layout" links have changed to not have "react" at the end of them - so just remove the last "react" from each url and you should be good there!

For example, it should be:  [Grid] (/layouts/grid)